server.js

```diff
@@ -1,3 +1,15 @@
 import app from './app.js';
+import { connectToDB } from './connections.js';
 
-app.listen({ port: process.env.PORT || 3000 });
+function loaders() {
+  try {
+    connectToDB();
```

Comment:
this is async, and you want to await it :)

Reply:
fixed